src/components/create-study-form.js

```diff
@@ -398,33 +421,15 @@ export const CreateStudyForm = ({ open, onClose }) => {
                             margin="dense"
                             value={studyName}
                             type="text"
-                            error={studyInvalid}
+                            error={
```

Comment:
Maybe we should add an error props for equivalent TextFileds component in create-filter-form.js and create-contingency-form.js 
Maybe other props as well, like autoFocus, margin, type, style, value ? and remove defaultValue elsewhere ?